lumigator/python/mzai/backend/backend/settings.py

```diff
@@ -118,5 +121,19 @@ def RAY_JOBS_URL(self) -> str:  # noqa: N802
     def SQLALCHEMY_DATABASE_URL(self) -> URL:  # noqa: N802
         return make_url(os.environ.get("SQLALCHEMY_DATABASE_URL", None))
 
+    @computed_field
+    @property
+    def API_CORS_ALLOWED_ORIGINS(self) -> list[str]:  # noqa: N802
+        """Retrieves the (comma separated) list of allowed origin URLs for CORS requests
+        against the Lumigator backend API.
+
+        Defaults to "*" (allows all) when no environment variable is set.
+        """
+        origins = self._api_cors_allowed_origins.strip()
+        if origins == "":
+            return ["*"]
```

Comment:
Just raising a concern here about defaulting to * when nothing is provided. WDYT about providing "localhost" (the strictest setting) by default and require users to provide a different value if the setup changes?

Reply:
I understand the hesitancy, I've made some changes so the defaults are configured in the `.env` file now (and no longer use `*`)